docs/installation/gateways.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Swarm Gateways
+id: gateway
+---
+
+For now, not all of your devices, friends and users of your app will be able to run a Bee node which is able to connect directly to the network using *libp2p*. 
+
+To access the Swarm network in these situations, it may be useful for you to provide a Swarm Gateway, which enables devices with only access to traditional web2 TCP/HTTP connections access to some elements of the swarm.
```

Comment:
```suggestion
To access the Swarm network in these situations, it may be useful for you to provide a gateway, which enables devices with access to only traditional web2 TCP/HTTP connections to access some elements of the swarm.
```